scripts/nlp_language_modeling/sft/data_clean.py

```diff
@@ -41,7 +41,7 @@ def data_clean(
     )
     if library == 'huggingface':
         tokenizer.add_special_tokens({'additional_special_tokens': ['<extra_id_0>', '<extra_id_1>', '<extra_id_2>']})
-    d = GPTSFTChatDataset(dataset_file, tokenizer, seq_len, 1)
+    d = GPTSFTChatDataset(dataset_file, tokenizer, seq_len, 1, hf_dataset=True)
```

Comment:
can we change the default back to False?
I dont think we should force hf_dataset, people can set it to True if they want to use it

Reply:
the problem is hf_dataset False will fail because some bug in the `jsontextmap` dataset.  Set this to True is safer.

Reply:
why was it not there earlier?
might be because of the `mp.set_start_method("spawn", force=True)` ?

Reply:
don't know. but our old dataset is not reliable that's why I implemented the hf dataset.

Reply:
This is `data_clean.py` script. It won't affect the SFT `GPTSFTChatDataset` initialization which is default to be false. I don't see any risk here.

Reply:
ah, didnt see this was in a script 

looks good